### ISSUE TITLE
Portuguese language in duplicate on Weblate

### ISSUE THREAD

**Issue by hugok79** [lxqt/lxqt#1869]
There is no need to have the Portuguese language in duplicate on Weblate.
Just have the Portuguese language (pt)
You can remove the Portuguese language (pt_PT)
Portugal uses the language (pt) or (pt_PT) so there is no need to have both.
Leave only the Portuguese (pt) which is the most complete.

**Comment by stefonarch** [comment]
In which component did you find that pt_PT files? I  noticed some of them in https://github.com/lxqt/pcmanfm-qt/pull/1257/files .
There should be pt and pt_BR.

Will remove them, thanks.

**Comment by stefonarch** [comment]
Any idea how this happened?

**Comment by tsujan** [comment]
> Any idea how this happened for so many files?

It's simple: people don't seem to agree on this. I also have pt_PT in FeatherPad and FeatherNotes, beside pt_BR.

**Comment by stefonarch** [comment]
For some countries one is sure enough (it,de). 

I noticed  that Featherpad has half pt and half pt_PT - so removing pt_Pt from weblate is not an option I think.

Here I think it was a simple mistake when choosing your language to translate in the settings. 
Removal of some 0 translated could prevent this. It was sure some clicking time...

https://translate.lxqt-project.org/languages/

**Comment by tsujan** [comment]
> I noticed that Featherpad has half pt and half pt_PT - so removing pt_Pt from weblate is not an option I think.

There are two PRs for FeatherPad and FeatherNotes. Instead of merging them, I prefer to rename `pt_PT` to `pt` and change `language="pt_PT"` to `language="pt"` inside the files. I think that will be reflected by Weblate correctly; won't it?

**Comment by stefonarch** [comment]
Should work fine, I can update it manually.
You did not configure a webhook afaik, updates are made by a cronjob every 24 hours still.
Often there is also ru_RU ro_RO id_ID and th_TH and hose  not needed too.

**Comment by stefonarch** [comment]
Hm no, there could be merge conflicts, because there are both translation.

**Comment by tsujan** [comment]
> Hm no, there could be merge conflicts, because there are both translation.

No, I have only `pt_PT` in FeatherPad and FeatherNotes.

There are two PRs that want to make `pt` files *besides pt_PT files*. I'm not sure it's a good idea to merge them. A simple renaming will solve the problem in FeatherPad and FeatherNotes *if there is a problem at all*.

**Comment by stefonarch** [comment]
In weblate there are both versions.
https://translate.lxqt-project.org/projects/tsujan/featherpad/
https://translate.lxqt-project.org/projects/tsujan/feathernotes/
The Feathernotes  pt version is empty so here you can delete it and rename pt_PT > pt  (better IMHO for all pt users everywhere)

**Comment by tsujan** [comment]
> The Feathernotes pt version is empty

FeatherNotes doesn't have `pt`. There's [this PR](https://github.com/tsujan/FeatherNotes/pull/146), which I didn't merge.

I think `pt` is more reasonable than `pt_PT`. I'm preparing a FeatherPad PR for its code; will change `pt_PT` to `pt` in it too. Will do the same for FeatherNotes later.

**Comment by tsujan** [comment]
Oops, now I should merge https://github.com/tsujan/FeatherNotes/pull/146 manually because it includes Japanese too (it was only for `pt` before).

**Comment by stefonarch** [comment]
For feathernotes I could delete the empty pt file in weblate and push.

**Comment by tsujan** [comment]
> For feathernotes I could delete the empty pt file in weblate and push.

Please don't do it. I fixed it for FeatherPad and will do it for FeatherNotes too; a PR before that would make the job a little harder. Weblate shouldn't complain; it just needs to update its translations after commits.

**Comment by hugok79** [comment]
Leave only **(pt)** for Portuguese and **(pt_BR)** for Brazilian Portuguese.
Trust me,I know what I'm talking about.
I am the Portuguese translator of practically all the desktop environments "GNOME, Xfce, MATE, Pantheon, Deepin DE, Trinity Desktop, IceWM, etc...
I also translate hundreds of opensource programs and Linux distributions...
Ubuntu, Linux Mint, deepin, MX Linux, elementary OS, Manjaro, etc...
I work with all translations platforms such as Lanchpad, Weblate, Transifex, Crowdin, Zanata etc... and offline with Poedit or others programs.

In conclusion, Portugal have two languages codes, **pt** and **pt_PT** but there must be only one of them in a project.

**Comment by stefonarch** [comment]
> Leave only **(pt)** for Portuguese and **(pt_BR)** for Brazilian Portuguese.

Until yesterday evening exactly  this was the situation, there were none *_pt.PT.* files anywhere in the LXQt repositories.
Thanks for all.

**Comment by stefonarch** [comment]
> Weblate shouldn't complain; it just needs to update its translations after commits.

Got an alert from weblate for merging conflict, looks like @yan12125 fixed already
https://translate.lxqt-project.org/projects/tsujan/featherpad/#repository (scroll down)

**Comment by tsujan** [comment]
> Got an alert from weblate for merging conflict

Yes, because I rescued FP's incomplete Portuguese translation by renaming it and closed https://github.com/tsujan/FeatherPad/pull/460.

FN's Portuguese translation didn't need a rescue because it was empty (I wasn't aware of that before).

Anyway, now FP and FN use `pt` and don't have `pt_PT`. Of course, `pt_BR` isn't related to this story.

> Until yesterday evening exactly this was the situation, there were none*_pt_PT.*files anywhere in the LXQt repositories.

Strange! Who added them?

**Comment by stefonarch** [comment]
Look at the commit history...

> Strange! Who added them?

https://github.com/lxqt/lxqt-panel/pull/1502/commits/5832fbffeeb1aa069b11fced7b05a9b74fef14cb

**Comment by hugok79** [comment]
I was the one who added but only some components! There were already others created ... but then I realized that the language **pt** already existed and as such there was no need for **pt** and **pt_PT**.

**Comment by tsujan** [comment]
> Maybe there is reason that adding translation is disabled bu default, will think about it.

Contacting admins should be required, IMHO.

**Comment by tsujan** [comment]
Completely unrelated to all this but I noticed it recently:

A CPU-consuming animation has been added to to https://github.com/. Whenever I sign out, 11% to 30% of the CPU time is taken by the browser (Firefox, Falkon and Chromium). Very nice!

**Comment by hugok79** [comment]
There is a dark mode option also in github now.
